docs/source/appendix/rtlil_text.rst

```diff
@@ -262,8 +262,8 @@ Switches
 Switches test a signal for equality against a list of cases. Each case specifies
 a comma-separated list of signals to check against. If there are no signals in
 the list, then the case is the default case. The body of a case consists of zero
-or more switches and assignments. Both switches and cases may have zero or more
-attributes.
+or more assignments followed by zero or more switches. Both switches and cases
```

Comment:
Just to make sure, not following this currently raises a warning with the intention that it will be enforced later?

Reply:
Yes exactly